js/script.js

```diff
@@ -0,0 +1,269 @@
+  rejectedCounter: 0,
+  items: ['whiteSneakers', 'blackSkinnyJeans'],
+  tags: ['sport', 'classic']
+},
+{
+  outfitId: 5,
+  imgSrc: 'images/5.jpg',
+  rank: 0,
+  rejectedCounter: 0,
+  items: ['whiteSneakers', 'blackSkinnyJeans'],
+  tags: ['sport', 'classic']
+},
+{
+  outfitId: 6,
+  imgSrc: 'images/6.jpg',
+  rank: 0,
+  rejectedCounter: 0,
+  items: ['whiteSneakers', 'blackSkinnyJeans'],
+  tags: ['sport', 'classic']
+},
+{
+  outfitId: 7,
+  imgSrc: 'images/7.jpg',
+  rank: 0,
+  rejectedCounter: 0,
+  items: ['whiteSneakers', 'blackSkinnyJeans'],
+  tags: ['sport', 'classic']
+},
+{
+  outfitId: 8,
+  imgSrc: 'images/8.jpg',
+  rank: 0,
+  rejectedCounter: 0,
+  items: ['whiteSneakers', 'blackSkinnyJeans'],
+  tags: ['sport', 'classic']
+},
+{
+  outfitId: 9,
+  imgSrc: 'images/9.jpg',
+  rank: 0,
+  rejectedCounter: 0,
+  items: ['whiteSneakers', 'blackSkinnyJeans'],
+  tags: ['sport', 'classic']
+},
+{
+  outfitId: 10,
+  imgSrc: 'images/10.jpg',
+  rank: 0,
+  rejectedCounter: 0,
+  items: ['whiteSneakers', 'blackSkinnyJeans'],
+  tags: ['sport', 'classic']
+},
+{
+  outfitId: 11,
+  imgSrc: 'images/11.jpg',
+  rank: 0,
+  rejectedCounter: 0,
+  items: ['whiteSneakers', 'blackSkinnyJeans'],
+  tags: ['sport', 'classic']
+},
+{
+  outfitId: 12,
+  imgSrc: 'images/12.jpg',
+  rank: 0,
+  rejectedCounter: 0,
+  items: ['whiteSneakers', 'blackSkinnyJeans'],
+  tags: ['sport', 'classic']
+}
+];
+
+let l = 0;
+let r = 1;
+const main = document.getElementById('main');
+function displayQuiz() {
+ quizDiv = document.createElement("div");
+ quizDiv.setAttribute("class", "quiz");
+if (l <= 5 && r <= 5) {
+  main.innerHTML = `
+  <h1 class="mainHeader">What would you prefer?</h1>
+  <div class="quiz">
+    <div id="leftPic" class="picWrapper">
+      <a>
+        <img src=${outfits[l].imgSrc} class="image">
+      </a>
+      <div class="middle">
+        <div class="text">❤️</div>
+      </div>
+    </div>
+    <div id="spliter" class="splitter">
+    VS
+    </div>
+    <div id="rightPic" class="picWrapper">
+      <a>
+        <img src=${outfits[r].imgSrc} class="image">
+      </a>
+      <div class="middle">
+        <div class="text">❤️</div>
+      </div>
+    </div>
+  </div>
+  `;
+} else {
+  result();
+
+  resultLeft = document.getElementById("leftRes");
+  resultCenter = document.getElementById("centerRes");
+  resultRight = document.getElementById("rightRes"); 
+  
+  resultLeft.addEventListener('click', resultL); 
+  resultCenter.addEventListener('click', resultC); 
+  resultRight.addEventListener('click', resultR);
+}
+
+voteRight = document.getElementById("rightPic");
+voteRight.addEventListener('click', voteCounterRight);
+voteLeft = document.getElementById("leftPic");
+voteLeft.addEventListener('click', voteCounterLeft);
+};
+
+const startBtn = document.getElementById('start');
+startBtn.addEventListener('click', displayQuiz);
+
+function voteCounterLeft() {
+if (r == l - 1) {
+  r = r + 2;
+  displayQuiz();
+} else
+  r++;
```

Comment:
Don't omit the brackets! this could lead to a potential errors which would be hard to spot.

Reply:
I lost them) Fixed!